From 2a87f70af8f3d2cf75c401fc0f6d06f398863c54 Mon Sep 17 00:00:00 2001
From: Tan Jui Nee <jui.nee.tan@intel.com>
Date: Tue, 28 Jun 2016 01:28:18 +0000
Subject: [PATCH 0097/2508] pinctrl: intel: use 'bool' state for
 PINCTRL_APL_DEVICE in Kconfig

commit 0fb3e1a90e4163e8f25f876eeee59670f19dbad4 from
https://github.com/01org/linux-apollolake-i

Intel Apollo Lake GPIO pin control Platform Device Emulation is only
working as build-in module.

Signed-off-by: Tan Jui Nee <jui.nee.tan@intel.com>
Signed-off-by: Bruce Ashfield <bruce.ashfield@windriver.com>
---
 drivers/pinctrl/intel/Kconfig |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/drivers/pinctrl/intel/Kconfig b/drivers/pinctrl/intel/Kconfig
index bab8823..417fb39 100644
--- a/drivers/pinctrl/intel/Kconfig
+++ b/drivers/pinctrl/intel/Kconfig
@@ -42,7 +42,7 @@ config PINCTRL_BROXTON
 	  configuring of SoC pins and using them as GPIOs.
 
 config PINCTRL_APL_DEVICE
-	tristate "Intel Apollo Lake GPIO pin control Platform Device Emulation"
+	bool "Intel Apollo Lake GPIO pin control Platform Device Emulation"
 	depends on PINCTRL_BROXTON
 	help
 	  This driver is to set up platform device in the absent of ACPI
-- 
1.7.5.4

